src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/integration/PrisonApiMockServer.kt

```diff
@@ -660,4 +663,49 @@ class PrisonApiMockServer : WireMockServer(9005) {
         )
     )
   }
+
+  fun stubGetMovementSuccess(agencyId: String, fromDate: LocalDateTime, toDate: LocalDateTime) {
+    stubFor(
+      get("/api/movements/$agencyId/in?fromDateTime=$fromDate&toDateTime=$toDate")
```

Comment:
just checking whether the dates need formatting? rather than just using the tostring

Reply:
[Looks like](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html#toString--) maybe the tostring on date won't include seconds if the seconds are 00?

Reply:
sorted